index.html

```diff
@@ -1,279 +1,274 @@
-              <div class="skill_display_Skills">
-                <img class="skill_display_rectangle57" src="images/Rectangle 57.png" />
-                <p class="skill_display_title3">Skills</p>
-              </div>
-              <ul class="skill_item_list3">
-                <li>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-              </ul>
-            </li>
-          </ul>
+
+        <hr class="hr-line_break">
+        
+        <div class="Language_skill">
+            <ul class="Language_skill_list">
+                <li class="Language_skill_bullet "><img class="border_line" src="./images/Rectangle55.png" alt="rectangle_icon">Language
+                </li>
+                <li class="language">Javascript</li>
+                <li class="language">Ruby</li>
+                <li class="language">Html</li>
+                <li class="language">Css</li>
+            </ul>
+            <ul class="Language_skill_list2">
+                <li class="Language_skill_bullet"><img class="border_line" src="./images/Rectangle56.png" alt="rectangle_icon">Frameworks
+                </li>
+                <li class="language">Bootstrap</li>
+                <li class="language">Ruby on Rails</li>
+                <li class="language">RSPec</li>
+                <li class="language">CapyBara</li>
+                <li class="language">Selenium</li>
+            </ul>
+            <ul class="Language_skill_list3">
+                <li class="Language_skill_bullet"><img class="border_line" src="./images/Rectangle57.png" alt="circle_icon">Skills</li>
+                <li class="language">Codekit</li>
+                <li class="language">Github</li>
+                <li class="language">Codepen</li>
+                <li class="language">Gitlab</li>
+                <li class="language">Terminal</li>
+            </ul>
         </div>
     </section>
-    <section>
-      <div class="formpage_content_container">
-         <p class="formpage_content"><strong>
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-          </strong>
-        </p>
-      </div>
-        <form class="contact_me_form"
-          action="https://formspree.io/f/xoqbwpby"
-          method="POST">
-          <label for="name" hidden>Name</label>
-          <input
-            type="text"
-            name="Name"
-            id="name"
-            class="inputs"
-            placeholder="Full names here"
-            maxlength="50"
-            required
-          /><br>
-          <label for="email" hidden>Email</label>
-          <input
-            type="email"
-            name="Email"
-            id="email"
-            class="inputs"
-            placeholder="abcd@email.com"
-            required
-          />
-          <label for="message" hidden>Message</label>
```

Comment:
- [ ] I suggest you remove the label from your form section. because it mutates the input form. kindly fix it.

Reply:
- Though you have hidden the label requested to be removed by the previous reviewer, note that it is important to have only what is needed in your HTML structure, this helps to make your page more maintainable and easy to read. Please fix this 👍 👍